adjust_data.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import fileinput
+
+parser = argparse.ArgumentParser(
+    description="Adjust a line from wb-cli so it can be fed into the target wiki"
+)
+
+parser.add_argument("-s", "--source", action="store", dest="source", required=True)
```

Comment:
Thinking about it again, we probably don't even need the target languages as we implicitly get these when requesting the entities. The for loop will filter what's not to be passed, so no need to intersect the sets.